cmd/controller/controller.go

```diff
@@ -45,24 +49,46 @@ func init() {
 	if importerImage == "" {
 		importerImage = IMPORTER_DEFAULT_IMAGE
 	}
-	pullPolicy = IMPORTER_DEFAULT_PULL_POLICY
+	importerPullPolicy = IMPORTER_DEFAULT_PULL_POLICY
 	if pp := os.Getenv(IMPORTER_PULL_POLICY); len(pp) != 0 {
-		pullPolicy = pp
+		importerPullPolicy = pp
+	}
+	
+	clonerImage = os.Getenv(CLONER_IMAGE)
+	if clonerImage == "" {
+		clonerImage = CLONER_DEFAULT_IMAGE
+	}
+	clonerPullPolicy = CLONER_DEFAULT_PULL_POLICY
+	if pp := os.Getenv(CLONER_PULL_POLICY); len(pp) != 0 {
+		clonerPullPolicy = pp
 	}
 
 	// get the verbose level so it can be passed to the importer pod
 	defVerbose := fmt.Sprintf("%d", IMPORTER_DEFAULT_VERBOSE) // note flag values are strings
-	verbose = defVerbose
+	importerVerbose = defVerbose
 	// visit actual flags passed in and if passed check -v and set verbose
 	flag.Visit(func(f *flag.Flag) {
 		if f.Name == "v" {
-			verbose = f.Value.String()
+			importerVerbose = f.Value.String()
 		}
 	})
-	if verbose == defVerbose {
+	if importerVerbose == defVerbose {
 		glog.V(Vuser).Infof("Note: increase the -v level in the controller deployment for more detailed logging, eg. -v=%d or -v=%d\n", Vadmin, Vdebug)
 	}
-
+	
+	// get the verbose level so it can be passed to the clone pods
+	defVerbose = fmt.Sprintf("%d", CLONER_DEFAULT_VERBOSE) // note flag values are strings
+	clonerVerbose = defVerbose
+	// visit actual flags passed in and if passed check -v and set verbose
+	flag.Visit(func(f *flag.Flag) {
```

Comment:
if clone verbose remains separate then I think you can refactor to use just the one `for` loop.